### PR TITLE
Hero Page Added in Home Folder

### PR DESCRIPTION
**Pull request by dhruvjaink07**
@CODEX108 @anaskhan28

### CODE REVIEW

**Comment by CODEX108** [comment]
@dhruvjaink07 pull the latest code from `home-page` branch and resolve the merge conflicts
You can refer the Navbar code and make changes from your side that so that nothing breaks

**Comment by dhruvjaink07** [comment]
@CODEX108 @anaskhan28 Can, you review the Pull Request ?

image

File: src/App.jsx [hunk 1]
```diff
@@ -6,6 +6,11 @@ function App() {
 
 
   return (
+
```
Comment (CODEX108) [inline src/App.jsx]: There can only be one `div` or `fragment` present inside the return statement i.e:

```js
return(
<>
</>
}
```

### OR
```js
return(
<div>
</div>
}
```

Next the other children inside this div

Comment (dhruvjaink07) [reply src/App.jsx]: Done! Thanks For Helping @CODEX108

**Pull request merged**